tests/server/api/test_server.py

```diff
@@ -74,7 +76,7 @@ async def raise_other_error():
             Exception,
         )
 
-    app = create_app(ephemeral=ephemeral)
+    app = create_app(ephemeral=ephemeral, ignore_cache=True)
```

Comment:
Uh yeah if you don't do this the test routes get added to the cached app and other tests see it :D